daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/TypeCalculator.scala

```diff
@@ -412,3 +451,83 @@ object TypeCalculatorCompiler {
   }
 
 }
+
```

Comment:
The below block is a modified version of the RangeBound file we removed above.

Instead of having a type parameter, it assumes the bound is a DataValuePrimitive.

This is a good example of why we might want a richer DataValue hierarchy. In theory we could declare this with DataValueNumeric instead. I am aware of no reason we cannot do this, but in practice, declaring new DataValue types seems to be very finicky work, so I do not think it is worth the benifit here.

Also, in practice RangeBound only every uses JBigInt at the moment.

Reply:
Put above comment into the code please.